### PR TITLE
chore(prow/jobs/pingcap/tidb): enable postsubmits job `e2e-test`

### PR DESCRIPTION
**Pull request by wuhuizuo**
but skip report currently for debuging

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

### CODE REVIEW

**Comment by wuhuizuo** [comment]
depend on https://github.com/PingCAP-QE/ci/pull/1674

File: prow/jobs/pingcap/tidb/tidb-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+postsubmits:
+  pingcap/tidb:
+    - name: pingcap/tidb/merged_e2e_test
+      agent: jenkins
+      decorate: false # need add this.
+      context: wip-e2e-test
+      skip_report: true
```
Comment (wuhuizuo) [inline prow/jobs/pingcap/tidb/tidb-postsubmits.yaml:7]: when the job ready, please change those lines

you can view the jobs at https://prow.tidb.net/?repo=pingcap%2Ftidb&type=postsubmit

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**